10.1/Dockerfile.fedora

```diff
@@ -19,14 +19,12 @@ LABEL summary="MariaDB is a multi-user, multi-threaded SQL database server" \
       io.k8s.description="MariaDB is a multi-user, multi-threaded SQL database server" \
       io.k8s.display-name="MariaDB 10.1" \
       io.openshift.expose-services="3306:mysql" \
-      io.openshift.tags="database,mysql,mariadb,mariadb101,galera"
-
-ENV NAME=mariadb VERSION=10.1 RELEASE=1 ARCH=x86_64
-LABEL BZComponent="$NAME" \
-        Name="$FGC/$NAME" \
-        Version="$VERSION" \
-        Release="$RELEASE.$DISTTAG" \
-        Architecture="$ARCH"
+      io.openshift.tags="database,mysql,mariadb,mariadb101,galera" \
+      BZComponent="$NAME" \
```

Comment:
We can use the lower-case variant for Fedora dockerfile as well now (https://fedoraproject.org/wiki/Container:Guidelines#LABELS).